server/api/binding.go

```diff
@@ -17,11 +17,11 @@ type Binding struct {
 	// For PostMenu, ChannelHeader locations specifies the icon.
 	Icon string `json:"icon,omitempty"`
 
-	// Label is the (usually short) primary text to display at the location.
+	// Name is the (usually short) primary text to display at the location.
```

Comment:
0/5. I chose `Label` because `Name` is used in `Field` as the JSON field (ID). Also, if we rename `Label`->`Name` here we should rename `Field.AutocompleteLabel` accordingly (`Field.AutocompleteFlagName`?) since it means the same thing. FWIW, I _like_ label since it's more indicative of the "Display Name" nature of it to me.

Reply:
Yeah `Label` sounds good